apps/myaccount/src/constants/profile-constants.ts

```diff
@@ -42,6 +42,22 @@ export class ProfileConstants {
 
     public static readonly USERNAME_CLAIM_NAME: string = "userName";
     public static readonly MOBILE: string = "mobile";
+
+    /**
+     * Account state constants.
+     */
+    public static readonly ACCOUNT_STATE_PENDING_SR: string = "PENDING_SR";
+
+    /**
+     * Preferred channel constants.
+     */
+    public static readonly PREFERRED_CHANNEL_EMAIL: string = "EMAIL";
+
+    /**
+     * SCIM schema constants.
+     */
+    public static readonly ACCOUNT_STATE: string = "ACCOUNT_STATE";
+    public static readonly PREFERRED_CHANNEL: string = "PREFERRED_CHANNEL";
```

Comment:
shall we add the doc comment for this as well?

Reply:
The `SCIM schema constants.` is for both `ACCOUNT_STATE ` and `PREFERRED_CHANNEL `